spec/factories/tasks.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :task do
+    title { "test" }
+    status { "todo" }
```

Comment:
enumの値は文字列のStringではなく、Symbol形式で記載した方が良いですね。ただの文字列を登録できるカラムでなく、enumで定義されているカラムであることを明示できます。
```suggestion
    status { :todo }
```